content/en/docs/refguide/modeling/application-logic/workflows/workflow-engine.md

```diff
@@ -136,6 +136,14 @@ All microflows that are run as part of the workflow are executed as an asynchron
 
 Failed workflows can be retried using the **Retry workflow** option of the [Change Workflow State microflow activity](/refguide/change-workflow-state/#operation). This option will attempt to run the user task from the point it failed. When the user task failed because no users were targeted, it is possible to manually correct user targeting and then use the **Retry Workflow** option to set the workflow into the in-progress state again.
 
+#### Timer Boundary Events {#timer-boundary-events}
+
+A timer boundary event is attached to the boundary of an activity. Once the activity begins, also the boundary timer events are initialized. When the set timer expires, the workflow engine will check if the parent activity is inProgress. If the parent activity has already been completed, aborted, or has failed, the trigger will be ignored and the boundary path will not be executed. If the workflow is paused or in an incompatible state, the boundary event path will commence its execution once the workflow resumes an in-progress state. Otherwise, if the activity is still ongoing, the boundary event path will initiate its execution.
```

Comment:
```suggestion
A timer boundary event is attached to the boundary of an activity. Once the activity begins, the boundary timer events are initialized. When the set timer expires, the workflow engine will check if the parent activity is inProgress. If the parent activity has already been completed, aborted, or has failed, the trigger will be ignored and the boundary path will not be executed. If the workflow is paused or in an incompatible state, the boundary event path will commence its execution once the workflow resumes an in-progress state. Otherwise, if the activity is still ongoing, the boundary event path will initiate its execution.
```

Reply:
I changed up this line so the suggested changes don't apply anymore.